### PR TITLE
Remove sidebar experiment.

### PR DESCRIPTION
**Pull request by sriramkrish85**
Sidebar is already live and out of experimental via the config file - so clean up the experiment in the code.

### CODE REVIEW

File: extensions/amp-sidebar/amp-sidebar.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ limitations under the License.
   </tr>
   <tr>
     <td width="40%"><strong>Examples</strong></td>
-    <td>None</td>
+    <td>[amp-sidebar.html](https://ampbyexample.com/components/amp-sidebar/)</td>
```
Comment (erwinmombay) [inline extensions/amp-sidebar/amp-sidebar.md]: incorrect markdown syntax

Comment (erwinmombay) [inline extensions/amp-sidebar/amp-sidebar.md]: sorry meant, it should be an html link here.

**Comment by erwinmombay** [comment]
LGTM after link fix

**Comment by sriramkrish85** [comment]
cc @sebastianbenz

**Pull request merged**